client/src/pages/Dashboard.tsx

```diff
@@ -1,11 +1,104 @@
-import React from 'react';
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import content from 'static/mock/dashboard';
+import { View, VulnerabilityViewColumn, Risk } from 'helpers/enums/dashboard';
+
+import QuickStats from 'components/QuickStats';
+import ViewButtons from 'components/ViewButtons';
+import SearchAndFilterBar from 'components/SearchAndFilter';
+import ShowInfoAndExport from 'components/ShowInfoAndExport';
+import PaginatedTable from 'components/PaginatedTable';
+
+//TODO: GET THE DATA FROM AN API CALL LATER ON - HAVE IT HANDLED BY A FUNCTION THAT'S REFERENCED IN THIS COMPONENT
```

Comment:
make issue - can you tag me in it too?
I'll be working on dashboard back end this week so we should work together a bit on those!